src/pages/settings.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import PageTemplate from "@/components/templates/PageTemplate";
+import SettingsRead from "@/components/organisms/SettingsRead";
+import SettingsChange from "@/components/organisms/SettingsChange";
+import SettingsDropDown from "@/components/organisms/SettingsDropDown";
+import api from "../lib/axios_settings";
```

Comment:
Please change `"../lib/axios_settings"` to `"@/lib/axios_settings"`

Reply:
Done